src/stylesheets/elements/_tags.scss

```diff
@@ -359,3 +359,32 @@ Label Title
 .sds-status--disabled {
   @include u-bg("disabled");
 }
+
+//Roadmap tags
+.label-done {
+  background-color: #70e17b;
+  color: #1b1b1b;
```

Comment:
Could we use sam-styles defined color names here instead of the hex codes? We might not have the exact matches with uswds, and in that case it's probably fine to pick the closest one. `#70e17b` looks like the hex for our primary color - https://federalist-0ad5a602-ca98-4a7e-8d6e-d9ece7bc4cf8.app.cloud.gov/preview/gsa/sam-styles/IAE-47383/components/detail/theme-color.html

Reply:
Updated the hex codes to make use of the colors already in sam-styles and created the more generic class name and extended it